sdk/openai/Azure.AI.OpenAI/src/Custom/ImageGenerationOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.AI.OpenAI
+{
+    /// <summary> Represents the request data used to generate images. </summary>
+    public partial class ImageGenerationOptions
+    {
+        /// <summary> Initializes a new instance of ImageGenerationOptions. </summary>
+        public ImageGenerationOptions()
+        {
+        }
```

Comment:
For consistency with other fields at the moment, yes. We have a pending review topic (that we didn't fully get to talk about last time) about how we want to treat property-only specification of required and conditionally required properties, and the outcome of that will reflect here and elsewhere.

That is, so long as we allow constructors for `ChatCompletionsOptions` without `messages` (unambiguously required request body parameter) or `CompletionsOptions` without `prompt` (conditionally required; non-Azure OpenAI will work without it), we should also allow an equivalent `ImageGenerationOptions` that can set its prompt via property rather than constructor parameter.

Reply:
Could we add this explanation as a comment on the constructor? It would be useful for future reference.

Reply:
Great idea. Added a comment to the body of each of these custom no-parameter constructors.